var/spack/repos/builtin/packages/extrae/package.py

```diff
@@ -35,54 +35,67 @@ class Extrae(Package):
        OpenMP, CUDA, OpenCL, pthread, OmpSs"""
     homepage = "https://tools.bsc.es/extrae"
     url      = "https://ftp.tools.bsc.es/extrae/extrae-3.4.1-src.tar.bz2"
+    version('3.7.1', sha256='95810b057f95e91bfc89813eb8bd320dfe40614fc8e98c63d95c5101c56dd213')
     version('3.4.1', '69001f5cfac46e445d61eeb567bc8844')
 
+    depends_on("autoconf", type='build')
+    depends_on("automake", type='build')
+    depends_on("libtool", type='build')
```

Comment:
`m4` may also be needed

Reply:
I think it is already pulled in as a runtime dependency of autoconf?  It is not called directly by this recipe at any point

Reply:
added. can we please get this merged?